__tests__/server/utils/logging/otel/__snapshots__/formatter.spec.js.snap

```diff
@@ -0,0 +1,157 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`otel formatter errors encodes ClientReportedError Error as parseable JSON 1`] = `
+Object {
+  "attributes": Object {
+    "agent": "Darwin x64",
+    "device.agent": "Browser/5.0 (compatible; WXYZ 100.0; Doors LQ 81.4; Boat/1.0)",
+    "error.message": "something broke",
+    "error.name": "ClientReportedError",
+    "error.stacktrace": "Error: something broke
+    at methodA (resource-a.js:1:1)
+    at methodB (resource-b.js:1:1)
+",
+    "request.address.uri": "https://example.com/page-the/error/occurred-on",
+    "request.metaData.code": "500",
+    "request.metaData.collectionMethod": "applicationError",
+    "request.metaData.correlationId": "123abc",
+    "request.metaData.moduleID": "healthy-frank",
+  },
+  "body": "something broke",
+  "severityNumber": 17,
+  "severityText": "ERROR",
+}
+`;
+
+exports[`otel formatter errors encodes Server Reported Error as parseable JSON with nested metadata objects 1`] = `
+Object {
+  "attributes": Object {
+    "agent": "Darwin x64",
+    "error.message": "something broke",
+    "error.name": "Error",
+    "error.stacktrace": "Error: something broke
+    at methodA (resource-a.js:1:1)
+    at methodB (resource-b.js:1:1)
+",
+    "metaData.moduleID": "healthy-frank",
+    "metaData.nestedObject.level1.level2.level3.level4": "logs nested objects correctly",
```

Comment:
Is this what we'd expect? These aren't exactly nested objects

Reply:
Yes this is the expected behavior. Nested objects are not supported.